Contents/Code/__init__.py

```diff
@@ -272,6 +272,13 @@ def search(self, results, media, lang, manual):
             search_helper.media.album
         )
 
+        #workaround for default search not getting results using Plex "Match" menu
+        #search_helper.media.album was "none", but search_helper.media.title was accurate
+        if not normalizedName or normalizedName == "None":
```

Comment:
Is this supposed to be the type `None`, or the literal string `"None"`?

Reply:
Literal string.  I tested as type first, but found it was being handled as a literal string when debugging the issue.  It's even capitalized, as well.  Here's a debug log snippet I was seeing when debugging.  This is what led me down the "let's try the title instead of the album if the album is "None" " approach.  Before this, every time I used the "Match" option (for albums that didn't get auto-matched for whatever reason), that Agatha Christie book and the ones after it was always the default set of search results.

```
2022-09-05 23:35:33,877 (700004cf5000) :  INFO (sandbox:19) - -----------------------------------ALBUM SEARCH-----------------------------------
2022-09-05 23:35:33,877 (700004cf5000) :  INFO (sandbox:19) - ID                  B008X8ND1O
2022-09-05 23:35:33,877 (700004cf5000) :  INFO (sandbox:19) - Title               Titan's Son
2022-09-05 23:35:33,877 (700004cf5000) :  INFO (sandbox:19) - ----------------------------------------------------------------------
2022-09-05 23:35:33,878 (700004cf5000) :  DEBUG (sandbox:19) - normalizedName = None
2022-09-05 23:35:33,878 (700004cf5000) :  DEBUG (sandbox:19) - chopping bracketed text = None
2022-09-05 23:35:33,878 (700004cf5000) :  DEBUG (sandbox:19) - normalizedName stripped = None
2022-09-05 23:35:33,878 (700004cf5000) :  INFO (sandbox:19) - -----------------------------------SEARCHING FOR "None"-----------------------------------
2022-09-05 23:35:33,880 (700004cf5000) :  DEBUG (networking:143) - Requesting 'https://api.audible.com/1.0/catalog/products?response_groups=contributors,product_desc,product_attrs&num_results=25&products_sort_by=Relevance&keywords=None'
2022-09-05 23:35:34,150 (700004cf5000) :  DEBUG (sandbox:19) - Found 25 result(s) for query "None"
2022-09-05 23:35:34,151 (700004cf5000) :  INFO (sandbox:19) - -----------------------------------Search results-----------------------------------
2022-09-05 23:35:34,155 (700004cf5000) :  ERROR (sandbox:19) - No album title found in file metadata
2022-09-05 23:35:34,155 (700004cf5000) :  INFO (sandbox:19) - Result #1
2022-09-05 23:35:34,156 (700004cf5000) :  INFO (sandbox:19) - ASIN is             B00ENI4SS0
2022-09-05 23:35:34,156 (700004cf5000) :  INFO (sandbox:19) - Author is           Agatha Christie
2022-09-05 23:35:34,156 (700004cf5000) :  INFO (sandbox:19) - Date is             2013-09-10
2022-09-05 23:35:34,156 (700004cf5000) :  INFO (sandbox:19) - Narrator is         Dan Stevens
2022-09-05 23:35:34,156 (700004cf5000) :  INFO (sandbox:19) - Score is            50
2022-09-05 23:35:34,157 (700004cf5000) :  INFO (sandbox:19) - Title is            And Then There Were None
2022-09-05 23:35:34,157 (700004cf5000) :  INFO (sandbox:19) - ----------------------------------------------------------------------
```

Reply:
Huh, interesting! I imagine the time you see this is when your file doesn't have `Album` set in the embedded data, and instead has `Title`. Good change anyhow.